app/scripts/lib/marketing-email-client.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A client to talk to the basket marketing email server
+ */
+
+'use strict';
+
+define([
+  'lib/xhr',
+  'lib/promise',
+  'lib/constants'
+], function (xhr, p, Constants) {
+
+  var NEWSLETTER_ID = Constants.MARKETING_NEWSLETTER_ID;
+
+  function MarketingEmailClient(options) {
+    options = options || {};
+
+    var self = this;
+
+    self._xhr = options.xhr || xhr;
+    self._baseUrl = options.baseUrl;
+  }
+
+  MarketingEmailClient.prototype = {
+    _request: function (path, type, accessToken, data, headers) {
```

Comment:
It will, I'll scope the token when I create it [over here](https://github.com/mozilla/fxa-content-server/blob/927e97f28b63ccbfb6e20eb8e56c46e753ba0f14/app/scripts/models/account.js#L126)